core-types/ints_and_floats_test.rb

```diff
@@ -2,24 +2,23 @@
 gem 'minitest', '~> 5.2'
 require 'minitest/autorun'
 require 'minitest/pride'
-require 'minitest/homework'
-
-class FixnumTest < Minitest::Homework
-  be_gentle!
 
+class FixnumTest < Minitest::Test
```

Comment:
Should we change the class name too? Looks like you renamed the file already, correct?